explore/src/main/scala/explore/config/sequence/GmosSequenceTable.scala

```diff
@@ -48,19 +59,31 @@ private sealed trait GmosSequenceTableBuilder[D: Eq]:
 
   private val AtomStepsColumnId: ColumnId = ColumnId("atomSteps")
 
-  private val columns: List[ColumnDef[SequenceTableRow, ?]] =
+  private def columns(
+    obsClass: ObserveClass,
+    sn:       Option[SignalToNoise]
+  ): List[ColumnDef[SequenceTableRow, ?]] =
     ColDef(
       AtomStepsColumnId,
       _.step.firstOf,
       header = " ",
       cell = _.value.map(drawBracket),
       size = 30.toPx
-    ) +: SequenceColumns.gmosColumns(ColDef, _.step.some, _.index.some)
+    ) +: SequenceColumns.gmosColumns(
+      ColDef,
+      _.step.some,
+      _.index.some,
+      r => {
+        val showScience = obsClass === ObserveClass.Science && r.flatMap(_.science).isDefined
+        val showAcq     = obsClass === ObserveClass.Acquisition && r.flatMap(_.fpuName).isEmpty
+        sn.filter(_ => showScience || showAcq)
+      }
```

Comment:
We need this logic in observe too. Can we place it in `lucuma-ui`?

Reply:
I'm not sure we need to show the S/N column in observe, do we?

Reply:
It appears in the inception document: 

<img width="751" alt="image" src="https://github.com/gemini-hlsw/explore/assets/1895643/c391932d-19ab-4f32-ba4e-cd26e1d8bea1">